addons/ofxOsc/src/ofxOscSender.h

```diff
@@ -1,91 +1,66 @@
-/*
- 
- Copyright (c) 2007-2009, Damian Stewart
- All rights reserved.
- 
- Redistribution and use in source and binary forms, with or without
- modification, are permitted provided that the following conditions are met:
- * Redistributions of source code must retain the above copyright
- notice, this list of conditions and the following disclaimer.
- * Redistributions in binary form must reproduce the above copyright
- notice, this list of conditions and the following disclaimer in the
- documentation and/or other materials provided with the distribution.
- * Neither the name of the developer nor the
- names of its contributors may be used to endorse or promote products
- derived from this software without specific prior written permission.
- 
- THIS SOFTWARE IS PROVIDED BY DAMIAN STEWART ''AS IS'' AND ANY
- EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
- WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
- DISCLAIMED. IN NO EVENT SHALL DAMIAN STEWART BE LIABLE FOR ANY
- DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
- (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
- LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
- ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
- SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
+// copyright (c) openFrameworks team 2010-2017
+// copyright (c) Damian Stewart 2007-2009
 #pragma once
 
-/**
-
-ofxOscSender
-
-an ofxOscSender sends messages to a single host/port
-
-*/
-
-namespace osc{
-	class UdpTransmitSocket;
-}
-#include <string>
 #include "OscTypes.h"
 #include "OscOutboundPacketStream.h"
 #include "UdpSocket.h"
 
 #include "ofxOscBundle.h"
-#include "ofxOscMessage.h"
 #include "ofParameter.h"
 #include "ofParameterGroup.h"
 
-
-class ofxOscSender
-{
+/// \class ofxOscSender
+/// \brief OSC message sender which sends to a specific host & port
+class ofxOscSender{
 public:
+
 	ofxOscSender();
 	ofxOscSender(const ofxOscSender & mom);
```

Comment:
now that emplace_back exists perhaps we could remove copy constructors from classes like this were they only existed to be able to put them in a vector (it doesn't make much sense to copy a class like this otherwise). but we can leave that for another moment